utils/test_results.py

```diff
@@ -359,16 +371,21 @@ def generate_test_results(
     if parameter is not None:
         match parameter:
             case GENERATE_TEST_RESULT_PARAM.FLAKY:
-                flakes = Flake.objects.filter(
-                    Q(repository_id=repoid)
-                    & (Q(end_date__date__isnull=True) | Q(end_date__date__gt=since))
-                )
+                totals = get_relevant_totals(repoid, branch, since)
 
-                flaky_test_ids = set([flake.test_id for flake in flakes])
+                flaky_test_ids = (
```

Comment:
being able to have a generalized pattern for all these makes it so much easier to read